atomic_reactor/utils/yum.py

```diff
@@ -44,7 +44,7 @@ def filename(self):
         partial sha256 checksum of the full repourl. This avoids multiple
         repos from being written to the same file.
         '''
-        urlpath = unquote(urlsplit(self.repourl, allow_fragments=False).path)
+        urlpath = unquote(urlsplit(self.repourl, allow_fragments=False).path).rstrip(os.sep)
         basename = os.path.basename(urlpath)
```

Comment:
Should we maybe raise an exception if basename is empty? (would only be possible with urls that don't have a path)

Reply:
we can raise